Doc/library/calendar.rst

```diff
@@ -138,21 +138,39 @@ interpreted as prescribed by the ISO 8601 standard.  Year 0 is 1 BC, year -1 is
 
    :class:`TextCalendar` instances have the following methods:
 
-   .. method:: formatmonth(theyear, themonth, w=0, l=0)
+   .. method:: formatweek(theweek, w=0, highlight_day=None)
+
+      Return a single week in a string with no newline. If *w* is provided, it
+      specifies the width of the date columns, which are centered. Depends
+      on the first weekday as specified in the constructor or set by the
+      :meth:`setfirstweekday` method.
+
+      .. versionchanged:: next
+         If *highlight_day* is given, this date is highlighted in color.
+         This can be :ref:`controlled using environment variables
+         <using-on-controlling-color>`.
```

Comment:
This should ideally be a separate PR, since we'd want to backport this doc addition.

Reply:
Please see PR https://github.com/python/cpython/pull/128353.